draft-ietf-dnsop-svcb-https.md

```diff
@@ -542,7 +549,7 @@ selectively dropping SVCB queries or responses, based on their size or
 other observable patterns.
 
 Similarly, if the client enforces DNSSEC validation on A/AAAA responses,
-it SHOULD NOT fall back to steps 5 or 6 if the SVCB
+it SHOULD NOT fall back to non-SVCB connection if a SVCB
```

Comment:
Should we clear on what it should do?  (eg, fail the connection?  or use the failed-to-validate SVCB response?   I assume the former, but a little ambiguous here.)

Reply:
OK, done